### ISSUE TITLE
label problem for meta-training stage

### ISSUE THREAD

**Issue by freeman-1995** [Sha-Lab/FEAT#46]
Hi thanks for the great work!

I have questions about the labels used in meta-training stage, I find you didn't use gt_label in your code.
you put gt_label in cuda(data, gt_label = [_.cuda() for _ in batch]), when computing cross_entropy, you use other labels(loss = F.cross_entropy(logits, label)), which is prepared by prepare_label func, this is very weird
